src/sentry/static/sentry/app/components/group/suggestedOwners.jsx

```diff
@@ -190,6 +209,9 @@ class SuggestedOwners extends React.Component {
             <GuideAnchor target="owners">
               <h6>
                 <span>{t('Ownership Rules')}</span>
+                <Hovercard body={this.getHovercardBody()}>
```

Comment:
For clarity, might be worth breaking out into `OwnershipRuleHovercard`, since `SuggestedOwnerHovercard` is also in this component

Reply:
Good point.  I'll definitely think about it for next time.  :)